zephyr/include/sof/lib/regions_mm.h

```diff
@@ -1,6 +1,6 @@
 /* SPDX-License-Identifier: BSD-3-Clause */
 /*
- * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
  *
  * Author: Jakub Dabek <jakub.dabek@intel.com>
```

Comment:
Tehnically this commit looks good! 

Anyhow, ...

There is something very upsetting about this commit which I cannot really pinpoint. Please use a consistent style throughout all the source code. 

The code is very hard to read, not to mention to understand. I broke down at `physical_mAPIng`. 

I will try to make some suggestion and maybe we can improve this commit.

Reply:
physical_mAPIng was clearly a mistake - not a reason to break down.